Yoast/wordpress-seo
# Added proper check for whether Premium is active or not.

## Context


* This PR ensures that the Fill (defined in Yoast SEO and utilized in the SEMrush modal) only loads if Premium is not activated.

## Summary


This PR can be summarized in the following changelog entry:

* Ensures the YoastRelatedKeyphrases fill for Free doesn't load in Premium.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Ensure that the Fill is properly loaded when opening the modal
* In `js/src/initializers/edit.js`, on line 227, invert the `isPremium` check.
* Rebuild the JavaScript
* Reload the page, reopen the modal.
* Ensure no Fill is loaded.

## UI changes

## Documentation

## Quality assurance
